GuildedRoseLLC/Details/DetailsViewController.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+
+public class DetailViewController: UIViewController {
+  
+    
```

Comment:
Request for Change: Remove extra space

Reply:
There is likely not time for this on this project, but in an ideal world, we would add a linter to the project.